### PR TITLE
Use new button styles

### PR DESCRIPTION
**Pull request by adriana-alter**
Goes with https://github.com/NYPL-Simplified/reusable-components/pull/16

Changes:

1) Styled (just left-aligned) Download CSV buttons
Screen Shot 2019-05-17 at 10 42 06 AM
Screen Shot 2019-05-17 at 10 41 55 AM

2) Styled Lists buttons
Screen Shot 2019-05-17 at 10 44 46 AM

3) Styled Lanes buttons
Screen Shot 2019-05-17 at 10 46 00 AM

4) Styled EditableConfigList buttons
Screen Shot 2019-05-17 at 10 47 00 AM

5) Styled InputList buttons
Screen Shot 2019-05-17 at 11 07 57 AM

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
1) This PR has all the Lane refactoring code as part of it. If I were to approve and you merged, those Lanes PR changes would be merged without approval (if the other one hasn't been approved yet). It'll be better if you point this PR to the `refactor-lanes` branch.
2) Doing `npm link` does not work for me... I get a lot of styling errors which didn't occur before:
Screen Shot 2019-05-20 at 9 38 13 AM

Screen Shot 2019-05-20 at 9 38 00 AM


Screen Shot 2019-05-20 at 9 37 49 AM

Screen Shot 2019-05-20 at 9 38 56 AM

**Comment by adriana-alter** [comment]
It is very weird that `npm link` isn't working for you; it's fine for me.  What happens when you do it?  Do the changes just not get picked up, or are there console errors, or...?

The issue with the white-on-white book details buttons isn't coming from this; the component is in the OPDS web client repo.  I already have a ticket for it here: https://jira.nypl.org/browse/SIMPLY-1976 . The other ones are addressed by the ReusableComponents branch (or would be if it would link properly for you!!).

**Comment by EdwinGuzman** [comment]
I double checked and `npm link` is working. The folder is referencing the local reusable components repo.

Now the only issue I see is:
Screen Shot 2019-05-20 at 9 37 49 AM

For the book page, I would fix that in this repo until it's fixed in the `opds-web-client` repo because it may take you longer to get that fixed and published and pulled into this repo. I'd be okay if you patch it here and later remove it.

**Comment by adriana-alter** [comment]
Screen Shot 2019-05-20 at 11 58 02 AM
Screen Shot 2019-05-20 at 11 57 53 AM

**Comment by EdwinGuzman** [comment]
Cool. I see the updates. I'm not sure if this is intentional but the "Edit name" and "delete" buttons are not the same size:
Screen Shot 2019-05-20 at 12 56 21 PM

**Comment by adriana-alter** [comment]
It is intentional that they're not the same size; "delete" is the same size as the other buttons in its group ("hide" and "save"), but "edit name" is part of a different section.  But if you think it looks weird, I'm fine with changing it!

**Comment by EdwinGuzman** [comment]
It's a bit weird just because when you click on "edit name" the "save name" button is bigger than the "edit name" button. This PR looks fine but what I mentioned before hasn't been done yet:

> This PR has all the Lane refactoring code as part of it. If I were to approve and you merged, those Lanes PR changes would be merged without approval (if the other one hasn't been approved yet). It'll be better if you point this PR to the refactor-lanes branch.

You should also update this branch with that branch.

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "bootstrap": "^3.3.6",
     "font-awesome": "^4.6.3",
     "isomorphic-fetch": "^2.2.1",
-    "library-simplified-reusable-components": "1.3.2",
+    "library-simplified-reusable-components": "1.3.6",
```
Comment (EdwinGuzman) [inline package.json]: Update this to the latests.

File: src/stylesheets/lanes.scss [hunk 2]
```diff
@@ -56,21 +56,21 @@
...
-        svg {
-          fill: $pagetextcolor;
-          height: .7em;
-          width: .7em;
-        }
-        &:hover {
-          border-color: transparent;
-        }
-      }
+      // .btn.show-lane, .btn.hide-lane {
```
Comment (EdwinGuzman) [inline src/stylesheets/lanes.scss]: Remove this if it's not needed.

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "simplified-circulation-web",
-  "version": "0.2.3",
+  "version": "0.2.4",
```
Comment (EdwinGuzman) [inline package.json]: This looks like it needs to be updated with master, along with an update to the changelog.

**Pull request merged**